src/shared/common/forms/date.jsx

```diff
@@ -72,10 +67,8 @@ class DateInput extends Component {
                                 name={monthKey}
                                 type="number"
                                 pattern="[0-9]*"
-                                min="1"
```

Comment:
I wonder if we should leave these in -- do they add semantic value?
(To disable the HTML5 validation we'll need to pass `<form novalidate>`, but I'm not sure we're ready to do that yet?)